.github/workflows/config_options.yml

```diff
@@ -0,0 +1,98 @@
+name: "config-options"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+      - stable-*.*
+  schedule:
+    # Every day at 3:30 AM UTC
+    - cron: "30 3 * * *"
+
+jobs:
+  with-external-planarity-bliss:
+    name: "${{ matrix.bliss }} ${{ matrix.planarity }}"
+    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        planarity:
+          - "--with-external-planarity"
+          - ""
+        bliss:
+          - "--with-external-bliss"
+          - ""
+        exclude:
+          - planarity: ""
+            bliss: ""
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Install micromamba environment from environment.yml . . ."
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: digraphs
+          cache-environment: true
+          create-args: ${{ matrix.bliss && 'bliss' || '' }} ${{ matrix.planarity && 'planarity' || ''}}
+      - name: "Set environment variables . . ."
+        run: |
+          echo "PKG_CONFIG_PATH=$MAMBA_ROOT_PREFIX/envs/digraphs/lib/pkgconfig:$MAMBA_ROOT_PREFIX/envs/digraphs/share/pkgconfig" >> $GITHUB_ENV
+          echo "LD_LIBRARY_PATH=$MAMBA_ROOT_PREFIX/envs/digraphs/lib" >> $GITHUB_ENV
+          echo "CFLAGS=-I$MAMBA_ROOT_PREFIX/envs/digraphs/include" >> $GITHUB_ENV
+          echo "LDFLAGS=-L$MAMBA_ROOT_PREFIX/envs/digraphs/lib" >> $GITHUB_ENV
+      - name: "Install GAP and clone/compile necessary packages"
+        uses: gap-actions/setup-gap@v2
+        with:
+          # TODO: Should this have NautyTracesInterface included? Or is that the point of this test?
+          GAP_PKGS_TO_BUILD: "io orb profiling grape datastructures"
+          GAPBRANCH: stable-4.13
```

Comment:
As the comment says, has `NautyTracesInterface` been excluded purposefully?

Reply:
I don't think it was excluded on purpose, the package works with and without it, the tests test with and without it enabled.